Startup.cs

```diff
@@ -66,7 +66,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 {
                     endpoints
                         .MapControllerRoute(name: "default",
-                        pattern: "{controller}/{action=Index}/{id?}");
+                        pattern: "api/{controller}/{action=Index}/{id?}");
```

Comment:
I don't think this ended up working when we overrode the Route on the controller that all the controllers are currently doing, so let's delete this change to not cause confusion :)